x-pack/test/functional/apps/maps/sample_data.js

```diff
@@ -90,10 +90,6 @@ export default function ({ getPageObjects, getService, updateBaselines }) {
       await kibanaServer.uiSettings.update({
         [UI_SETTINGS.TIMEPICKER_QUICK_RANGES]: SAMPLE_DATA_RANGE,
       });
-      //running the rest of the tests with limited roles
```

Comment:
Why was security roles removed? I think that maybe we should add the required roles instead of removing.

Reply:
I think that was the problem with empty screenshots in tests. There's a number of error:
<img width="1979" alt="1" src="https://user-images.githubusercontent.com/3940730/159549148-1f18a055-4098-41f1-b2ae-128e89da0198.png">
<img width="1979" alt="2" src="https://user-images.githubusercontent.com/3940730/159549170-1cf17820-8e30-4bb6-bfa3-a74a081c2bc6.png">
Without these roles, the tests pass.

Still looking for why the resolution have changed...

Reply:
I've find out that in main branch the sample_data.js test is also failing on local environment. It has 500 error when accessing `tutorial_directory/sampleData` page after setting the specific roles like `['global_maps_all', 'kibana_sample_read']`